Source/interactivity_manager_impl.cpp

```diff
@@ -1229,18 +1246,23 @@ void interactivity_manager_impl::process_reply(const web::json::value& jsonReply
                 else
                 {
                     LOGS_INFO << L"Unhandled reply: " << jsonReply.serialize().c_str() << "to message " << message->to_string();
+                    messageWasProcessed = false;
                 }
             }
         }
         else
         {
             LOGS_INFO << "Unexpected json reply, no id";
+            messageWasProcessed = false;
         }
     }
     catch (std::exception e)
     {
         LOGS_ERROR << "Failed to parse service reply";
+        messageWasProcessed = false;

```

Comment:
>messageWasProcessed = false; [](start = 8, length = 28)

I feel like messageWasProcessed isn't the correct concept here. "isCustomMessage" might be more accurate. "messageWasProcessed" seems to be conflating two ideas - a well-formed but opaque message vs a malformed message.